tests/test_extension_throttle.py

```diff
@@ -103,56 +95,6 @@ def test_maxdelay_definition(value, expected):
     assert at.maxdelay == expected
 
 
-@pytest.mark.parametrize(
-    ("min_spider", "min_setting", "start_setting", "expected"),
-    [
-        (UNSET, UNSET, UNSET, AUTOTHROTTLE_START_DELAY),
-        (AUTOTHROTTLE_START_DELAY - 1.0, UNSET, UNSET, AUTOTHROTTLE_START_DELAY),
-        (AUTOTHROTTLE_START_DELAY + 1.0, UNSET, UNSET, AUTOTHROTTLE_START_DELAY + 1.0),
-        (UNSET, AUTOTHROTTLE_START_DELAY - 1.0, UNSET, AUTOTHROTTLE_START_DELAY),
-        (UNSET, AUTOTHROTTLE_START_DELAY + 1.0, UNSET, AUTOTHROTTLE_START_DELAY + 1.0),
-        (UNSET, UNSET, AUTOTHROTTLE_START_DELAY - 1.0, AUTOTHROTTLE_START_DELAY - 1.0),
-        (UNSET, UNSET, AUTOTHROTTLE_START_DELAY + 1.0, AUTOTHROTTLE_START_DELAY + 1.0),
-        (
-            AUTOTHROTTLE_START_DELAY + 1.0,
-            AUTOTHROTTLE_START_DELAY + 2.0,
-            UNSET,
-            AUTOTHROTTLE_START_DELAY + 1.0,
-        ),
-        (
-            AUTOTHROTTLE_START_DELAY + 2.0,
-            UNSET,
-            AUTOTHROTTLE_START_DELAY + 1.0,
-            AUTOTHROTTLE_START_DELAY + 2.0,
-        ),
-        (
-            AUTOTHROTTLE_START_DELAY + 1.0,
-            UNSET,
-            AUTOTHROTTLE_START_DELAY + 2.0,
-            AUTOTHROTTLE_START_DELAY + 2.0,
-        ),
-    ],
-)
-def test_startdelay_definition(min_spider, min_setting, start_setting, expected):
```

Comment:
Is this test useless/duplicate without `min_spider`?

Reply:
It seems, if you see the only assert is: `assert spider.download_delay == expected` lmk if you want to keep and drop the `min_spider` no sure what we would check in this case.

Reply:
OK, this is stupid :)

The extension sets `spider.download_delay` in `_spider_opened()` (based on `AUTOTHROTTLE_START_DELAY`, `DOWNLOAD_DELAY` and the original value of `spider.download_delay`, just like the params of this test), so that the downloader could use it as the start delay value for newly created slots in `scrapy.core.downloader._get_concurrency_delay()`. So if we want to stop using `spider.download_delay` we need this to work in some other way (and I don't think we can change the `DOWNLOAD_DELAY` setting value here).

@Gallaecio thoughts?

Reply:
Puff, headache incoming… :sweat:

Reply:
Maybe we should keep download_delay until https://github.com/scrapy/scrapy/issues/6660 is addressed?